### ISSUE TITLE
Movieset artwork cannot be saved into separate directories per set

### ISSUE THREAD

**Issue by rassaei** [Komet/MediaElch#1158]
**Is your feature request related to a problem? Please describe.**
Hi
I can set the following:
```
Movie Set Artwork: "Separate artwork directory"
Movie Set Poster Filename: "<setName>/poster.jpg"
Artwork directory "/foo/bar/moviesetartwork"
```
But when I select an image and save it, for e.g. the Movie Set named "Foo Bar", it just saves it as "/foo/bar/moviesetartwork/Foo Bar poster.jpg".
As Kodi19 now supports local Artwork for Sets but requires it to be in a separate directory per Set, this feature would be very helpful.

**Describe the solution you'd like**
Maybe the Movie Set Artwork should be saved in such a directory that I described by default. Or at least it should be an option. Here is the Kodi documentation about local movieset artwork: https://kodi.wiki/view/Movie_set_information_folder#Create_Movie_Set_Sub_Folders

**Describe alternatives you've considered**
Doing it manually and not using MediaElch is the only thing that comes to mind :(

**Additional context**
Kodi v19 Documentation: https://kodi.wiki/view/Movie_set_information_folder#Create_Movie_Set_Sub_Folders

**Comment by bugwelle** [comment]
I agree that MediaElch should support it. Thanks for creating this feature request. :-)
